cloudsync/tasks.py

```diff
@@ -1,16 +1,14 @@
 """
 Tasks for cloudsync app
 """
-
 import os
 import re
-from urllib.parse import unquote
-
-import boto3
+import urllib
```

Comment:
When I moved ```from urllib.parse import unquote``` after ```import boto3```, pylint complained that urllib had to be imported 1st. So I changed it to ```import urllib``` in order to keep all the ```from ``` statements after the ```import``` statements.